README.md

```diff
@@ -2,6 +2,7 @@
 
 [![Travis build status](https://travis-ci.org/flipkart-incubator/gojira.svg?branch=master)](https://travis-ci.org/flipkart-incubator/gojira) 
 [![Language grade: Java](https://img.shields.io/lgtm/grade/java/g/flipkart-incubator/gojira.svg)](https://lgtm.com/projects/g/flipkart-incubator/gojira/context:java)
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.flipkart/gojira-core/badge.svg)](https://maven-badges.herokuapp.com/maven-central/com.flipkart/gojira-core)
```

Comment:
we push to clojars. you may want to use: [![Clojars Project](https://img.shields.io/clojars/v/com.flipkart/gojira-core.svg)](https://clojars.org/com.flipkart/gojira-core) as is added in this page: https://clojars.org/com.flipkart/gojira-core